src/drt-theme.ts

```diff
@@ -75,12 +75,30 @@ const defaultValues = createTheme({
 
 const drtTheme = createTheme({
   palette: {
-    primary: defaultValues.palette.primary,
+    primary: {
```

Comment:
I don't know if these are ok or not. You need to check they don't have any adverse affects on our components